example/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
   flutter:
     sdk: flutter
 
+  #flutter_line_sdk: 1.2.4
```

Comment:
Not sure about this...Is it for some certain purpose or should we remove this?

Reply:
it's for verification. let me remove it.